src/js/structs/StateSummary.js

```diff
@@ -30,17 +28,27 @@ module.exports = class StateSummary {
       this.metadata.successfulSnapshot = options.successful;
     }
     this.metadata.date = options.date || Date.now();
+
+    let slaves = this.snapshot.slaves || [];
     // Store computed data – this is something we may not need to store
     this.metadata.slaveTotalResources = MesosSummaryUtil.sumResources(
       // We may only want to get the active slaves...
-      _.pluck(this.snapshot.slaves, 'resources')
+      slaves.map(function (slave) {
+        return slave.resources || null;
```

Comment:
we don't really need the `|| null` for all of these, because the original `_.pluck` will return `undefined` for them anyways...

plus, `slave.resources` might be something like an empty string, 0, etc. so I elect to just return `slave.resources` straight up (same for all of the pluck replaces)